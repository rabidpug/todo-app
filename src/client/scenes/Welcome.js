import HelloButton from 'Containers/HelloButton';
import HelloMessage from 'Containers/HelloMessage';
import React from 'react';

const Welcome = () => (
  <div>
    <HelloMessage />
    <HelloButton />
  </div>
);

export default Welcome;
