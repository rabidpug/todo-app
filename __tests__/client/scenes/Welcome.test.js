import React from 'react';
import Welcome from 'Scenes/Welcome';
import { shallow, } from 'enzyme';
import { shallowToJson, } from 'enzyme-to-json';

describe(
  'TopBar', () => {
    it(
      'Should render the TopBar', () => {
        const wrapper = shallow( <Welcome /> );

        expect( shallowToJson( wrapper ) ).toMatchSnapshot();
      }
    );
  }
);
