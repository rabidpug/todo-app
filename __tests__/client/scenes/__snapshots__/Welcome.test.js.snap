// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TopBar Should render the TopBar 1`] = `
<div>
  <Connect(Message) />
  <Connect(Button) />
</div>
`;
